enterprise/server/soci/soci.go

```diff
@@ -0,0 +1,316 @@
+package soci
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/commandutil"
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
```

Comment:
nit: move the aliased imports below into a separate group

Reply:
Done